application/domain/domain.go

```diff
@@ -75,6 +75,20 @@ const (
 	MessageTemplateRequestReceived                 = "request_received"
 )
 
+// User preferences
+const (
+	UserPreferenceKeyLanguage        = "language"
+	UserPreferenceLanguageEnglish    = "en"
+	UserPreferenceLanguageFrench     = "fr"
+	UserPreferenceLanguageSpanish    = "es"
+	UserPreferenceLanguageKorean     = "ko"
+	UserPreferenceLanguagePortuguese = "pt"
+
+	UserPreferenceKeyUnits      = "units"
+	UserPreferenceUnitsImperial = "imperial"
```

Comment:
```suggestion
	UserPreferenceUnitsUSCustomary = "uscs"
```
Since UK and Canada primarily use metric, US customary is probably more applicable.

Reply:
As long as the UI uses well known terminology.  I don't remember hearing of USCustomary before.